docs/metrics/common_values.md

```diff
@@ -0,0 +1,20 @@
+---
+comments: true
+status: new
+---
+
+# Common Values
```

Comment:
This header and description is not really descriptive :/ Why not keep it along with the rest of the metrics docs?